modules/indexing/src/test/java/org/apache/ignite/internal/processors/cache/distributed/near/IgniteCacheQueryNodeRestartSelfTest.java

```diff
@@ -193,10 +203,12 @@ protected IgniteInternalFuture createRestartAction(final AtomicBoolean done, fin
 
                     startGrid(idx);
 
-                    Thread.sleep(nodeLifeTime);
+                    resetBaselineTopology();
```

Comment:
You should avoid changing existing tests logic if it's not buggy.
They must pass as is.

Reply:
These tests count of quantity of events about stopping rebalance, but after this fix rebalance's events fires significant less often.